### ISSUE TITLE
Less does not produce any CSS

### ISSUE THREAD

**Issue by keenaudio** [sundialmusic/animaltronix#1]
If you copy the contents of "animaltronix.less" and paste it into the left-hand text area on http://less2css.org/ You will see that no CSS is generated.  My guess is that this .less file is not the complete less, there are some other files missing.  Maybe this file was included by another file?  Look around, I think you are missing something.

**Comment by keenaudio** [comment]
Looking at the HTML, seems you are using bootstrap.  Perhaps this file is meant to work in conjunction with the bootstrap less, setting variables and overrides, but otherwise compiling the bootstrap less to produce the css.

**Comment by sundialmusic** [comment]
Yes... this was the reason for using the less..  I'm obviously a complete
newb with less... so does this mean that your original idea of just using
scss would not be preferred since Bootstrap + Less is kind of a "known
framework"?

**Comment by keenaudio** [comment]
Bootstrap comes in both flavors: http://getbootstrap.com/

It was originally in less, but was ported to sass some time ago.  There are many ways to use it, it is definitely a "known framework", very popular at least for young projects.  As a project matures, it may eventually become an undesired overhead, but it certainly gets you off and running quickly (thus the name, "bootstrap").

There are many ways to add it into your project.  http://getbootstrap.com/getting-started/ You can get it with NPM, in which case it will end up in your node_modules directory.  There is another tool named Bower which is very similar to NPM except is designed for CLIENT-side resources (vs. Node which is designed for server/desktop modules).  However, there are many ways to bundle/build node modules so that they are useful on the client side.

Since you will be compiling the less (or sass) using gulp, you have complete control over both the source directory and the output directory.  So it would not really matter to you whether the bootstrap source was in node_modules/ or bower_components/.  Bootstrap is one of many modules that is available in both systems.  Certain modules are only available in one or the other, and that becomes a forcing function.  In this case, you can use whichever is most convenient:
- Download the source and add it anywhere in your project
- npm install bootstrap --save-dev  (will be in node_modules)
- npm install -g bower; bower install bootstrap --save  (will be in bower_components)

Since you already have other node modules and a package json, personally I would not go down the bower route (not just for bootstrap, there may be another reason to prefer bower) http://bower.io/  I would probably just install the bootstrap node module.   But then somehow the bootstrap less and your animaltronix less need to be stitched together.  There should be something in the old project source that indicates how it used to work.

**Comment by keenaudio** [comment]
I believe the code in animaltronix.less is a bootstrap "theme" http://antjanus.com/blog/uncategorized/make-bootstrap-3-theme-proper-way/

Were there some other lines in this file, like any @import statements? This would be one way the theme would be "stitched" together with the bootstrap source

**Comment by keenaudio** [comment]
The comment at the top says a lot: "Live LESS customizer"  http://bootstrap-live-customizer.com/

**Comment by keenaudio** [comment]
As you can see, "animaltronix.less" is just a customized version of "variables.less" from the bootstrap project: https://github.com/twbs/bootstrap/blob/master/less/variables.less

**Comment by keenaudio** [comment]
I believe this would work: http://stackoverflow.com/a/19514889

At the top of your "animaltronix.less" file, just @import the bootstrap.less

Variables are evaluated "lazily", and the LAST defined value takes precedent when it is compiled to CSS
